hibernate-core/src/main/java/org/hibernate/dialect/AbstractHANADialect.java

```diff
@@ -1648,4 +1648,9 @@ public boolean supportsNoWait() {
 	public boolean supportsJdbcConnectionLobCreation(DatabaseMetaData databaseMetaData) {
 		return false;
 	}
+
+	@Override
+	public String getNoColumnsInsertString() {
+		throw new MappingException( "SAP HANA requires at least one value in insert value-list clause." );
```

Comment:
@Naros I'm a bit annoyed by the fact that we only trigger the exception at runtime. I don't think it's worth adding a dialect method for such a corner case so I suppose we'll have to leave with it?